services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/directory/DiagnosisKeysDateDirectoryTest.java

```diff
@@ -124,14 +126,17 @@ void testDoesNotIncludeCurrentDateInDirectoryStructure() {
     runDateDistribution(diagnosisKeys, LocalDateTime.of(1970, 1, 5, 12, 0));
     Set<String> actualFiles = Helpers.getFilePaths(outputFile, outputFile.getAbsolutePath());
     Set<String> expectedDateAndHourFiles = getExpectedDateAndHourFiles(Map.of(
-        "1970-01-03", List.of("0", "1", "2", "3", "4"),
-        "1970-01-04", List.of("0", "1", "2", "3", "4"),
-        "1970-01-05", List.of("0", "1", "2", "3", "4")), "1970-01-05");
+        "1970-01-03", listOfHoursAsStrings(0, 23),
+        "1970-01-04", listOfHoursAsStrings(0, 23),
+        "1970-01-05", listOfHoursAsStrings(0, 11)), "1970-01-05");
     assertThat(actualFiles).isEqualTo(expectedDateAndHourFiles);
   }
 
   @Test
-  void testDoesNotIncludeEmptyDatesInDirectoryStructure() {
+  @Disabled("Temporarily disabling this test as part of the fix for issue #650."
```

Comment:
`testWhenDemoProfileIsActiveItDoesIncludeCurrentDateInDirectoryStructure` sets a configuration parameter but does not reset it to it's original value afterwards.

Reply:
Mitigation would be use a mock or spy instead.

Reply:
Will go for the option to revert the state of the configuration back to what it was before the testWhenDemoProfileIsActiveItDoesIncludeCurrentDateInDirectoryStructure started. In the context of running tests in parallel this could be a problem, but more refactoring would be needed anyway to support such test runs.